addons/website_slides/static/src/scss/slides_slide_fullscreen.scss

```diff
@@ -1,3 +1,6 @@
+#wrapwrap:has(div.o_wslides_fs_main) {
```

Comment:
Have to say the reviewer is not sure of what is exactly the issue, as this code does not exist anymore in the diff ... so hard to say :D .

Anyway I read words like "revamping the whole thing", which is generally not a good idea unless you have strong reasons it will be better afterwards.

If your issue is loading time of the page, would begin with classic sentence: is it slow currently, is it a real bottleneck, would we gain something ?

Reply:
It don't think it is slow :thinking:
The main fix that I needed to do was to remove the double scrollbars in the fullscreen display
![image](https://github.com/odoo/odoo/assets/107679666/81cc085a-900a-45f5-826e-5a3f66ae7564)
Apparently it is a fullscreen frame above an empty website page and the website page is overflowing the viewport.
I tried to add a `overflow-hidden` class on the website page to prevent showing the scrollbar as we don't see the website page in fullscreen mode anyway (we don't see the header or the footer). It seemed to work but I'm using the `has:` in the css which seems not recommended.

@julien-banken proposed to remove the empty website page as it is useless here and only load the fullscreen that way the second scrollbar will not be displayed and we will also reduce the loading time (it is always a plus).
But I don't know if it is doable as I assume all the `website_slides` module is displayed in a website page ? Also we're not sure if it should be done or if it is out of this task scope ?

What do you think? :slightly_smiling_face: 
Thank you!

Reply:
We have quite a lot of rules using  ``:has`` ... sorry for firefix but it seems a lot of rules would be broken.

Reply:
Ok, mostly snippets actually. Anyway not sure what is the best solution, not an expert in html stuff. Maybe see if design team has some ideas ?

Reply:
The bug doesn't appear on every screen but I found a way to reproduce by reducing the window height manually to force the overflow.

Runbot with the fix : https://36507064-saas-16-2-all.runbot171.odoo.com/slides/slide/tree-infographic-8?fullscreen=1
![image](https://github.com/odoo/odoo/assets/107679666/e32a990c-b396-4e3e-aa87-9aa6cd1d42c5)


saas-16.2 runbot to compare : https://44959837-saas-16-2-all.runbot138.odoo.com/slides/slide/tree-infographic-8?fullscreen=1
![image](https://github.com/odoo/odoo/assets/107679666/76cf5ff7-f5f5-4668-a8b1-303495f73532)

Reply:
I'm in contact with the design team we're looking into it :slightly_smiling_face:

Reply:
Hello @tde-banana-odoo :slightly_smiling_face: 
I looked into it with the design team.

My previous proposition using the `:has` was not a solution as it is prone to performance and compatibility issues.

What causes the issue is that, when the page is squashed (vertically and horizontally), the website page footer columns become full-width which is increasing its height and makes the second scrollbar appear. So the website page footer is triggering the overflow.

We tried to find a css fix. Using `position: fixed` instead of `absolute` on the `.o_wslides_fs_main` element fix the double scrollbar display but it also prevents the scrollbar from being scrolled when the mouse pointer is directly on it (when the pointer is on the slide and we use the mouse middle button to scroll it, it's ok). https://tinyurl.com/29vbq64j We don't know why this is happening but I think it is caused by the scrollbars superposition as scrolling actually triggers the scrollbar from the website page (the hidden one) and not the one from the slide.

We couldn't find a css fix that is correctly working even after much tries so I tried to fix it from the JS. As the footer is the composent causing the issue and it is not needed on the fullscreen view anyway, I just removed it from the view.
And It is working great :slightly_smiling_face: 

What do you think about it?
Thank you!

Reply:
If you can't tame it, kill it. Efficient and minimal diff :D .